activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/service/ModelService.java

```diff
@@ -103,8 +111,28 @@ public ModelService(ModelRepository modelRepository,
                                          pageable);
     }
 
+    public Model createNewModelObject(String type,
```

Comment:
`buildModel` ? `Create` gives me the idea of something that will be persisted as well.. `build` the idea that you are just initializing the jaavbean